--- origsrc/hitori-0.2.1/configure.ac	2008-07-08 11:38:09.000000000 -0500
+++ src/hitori-0.2.1/configure.ac	2008-07-08 23:08:29.046875000 -0500
@@ -39,7 +39,13 @@
 dnl Dependencies
 dnl ***************************************************************************
 
+PKG_PROG_PKG_CONFIG
+
+if `$PKG_CONFIG --atleast-version=2.13 gtk+-2.0`; then
 PKG_CHECK_MODULES(GENERAL, glib-2.0 gtk+-2.0 >= 2.13 gmodule-2.0 cairo >= 1.4)
+else
+PKG_CHECK_MODULES(GENERAL, glib-2.0 gtk+-2.0 >= 2.12 gmodule-2.0 cairo >= 1.4 libgnome-2.0)
+fi
 AC_SUBST(GENERAL_CFLAGS)
 AC_SUBST(GENERAL_LIBS)
 
--- origsrc/hitori-0.2.1/src/interface.c	2008-06-22 16:18:07.000000000 -0500
+++ src/hitori-0.2.1/src/interface.c	2008-07-08 23:08:29.093750000 -0500
@@ -21,6 +21,9 @@
 #include <cairo/cairo.h>
 #include <math.h>
 #include <glib/gi18n.h>
+#if !GTK_CHECK_VERSION(2, 13, 1)
+#include <libgnome/gnome-url.h>
+#endif
 
 #include "config.h"
 #include "main.h"
@@ -481,9 +484,13 @@
 
 	g_get_current_time (&current_time);
 
+#if GTK_CHECK_VERSION(2, 13, 1)
 	if (gtk_show_uri (gtk_widget_get_screen (hitori->window), "ghelp:hitori",
 			  current_time.tv_sec * 1000 + current_time.tv_usec,
 			  &error) == FALSE) {
+#else
+	if (gnome_url_show ("ghelp:hitori", &error) == FALSE) {
+#endif	  
 		GtkWidget *dialog = gtk_message_dialog_new (NULL,
 				GTK_DIALOG_MODAL,
 				GTK_MESSAGE_ERROR,
